doc/code_examples/Tutorial_ProForma.py

```
@@ -0,0 +1,245 @@
+        # Resolve modifications using ModificationsDB
+        oms.resolveModifications(pf)
+
+        # Convert to AASequence
+        aas = oms.toAASequence(pf, oms.AASequenceConversionPolicy.STRICT)
+        print(f"AASequence: {aas.toString()}")
+        print(f"Mono mass:  {aas.getMonoWeight():.4f} Da")
+
+    # Check conversion issues for complex peptidoforms
+    pf = oms.parse("[Phospho]?STYPEPTIDE")
+    issues = oms.getAASequenceConversionIssues(pf)
+    print(f"\nConversion issues for '[Phospho]?STYPEPTIDE':")
```

Ruff (0.14.13)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)